### PR TITLE
Spaces to tabs (resolve #26)

### LINKED ISSUE #26: Avoid mixing tabs and spaces for indentation
## Describe the bug

Indentation should either use tabs or spaces, but not both. The .eslintrc.js and .editroconfig files both define indentation as 2 tabs. However, there are files where there are spaces used for indentation, files that have some lines with spaces and others with tabs, and files that have both tabs and spaces used on the same line.

## To Reproduce

1. Open [.eslintrc.js](https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/master/.eslintrc.js#L20)
2. Notice the inconsistent indentations

On GitHub you'll notice this as the indentation being larger for some lines than others. In an editor you can set the invisible characters to be shown.

## Expected behavior

The indentations should all use a consistent format; character and size. 
_**Note:** you may want to have some file types use tabs and others spaces. That is acceptable but should be consistent. And this should be defined in the editorconfig and/or lint files._

### PR DESCRIPTION
**Pull request by TeddTech**
resolved issue #26 

The split view will probably be easier for looking over this PR

### CODE REVIEW

**Comment by jobara** [comment]
@TeddTech at least the .eslintrc file still contains a mix of spaces and tab characters for indentations. 

In looking a bit deeper into things, I think you may have a conflict between the vue linting and editor config settings. Is there a particular reason you chose to go with the 2 tabs for indentation?

The eslint rules for vue appear to default to 2 spaces.
- https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/script-indent.md
- https://eslint.vuejs.org/rules/html-indent.html

**Comment by greatislander** [comment]
> resolved issue #26

@TeddTech Please revisit https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords. In order for this pull request to automatically close the related issue, you need to write "resolved #26" (keyword immediately followed by issue number).

**Comment by TeddTech** [comment]
> https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

My mistake. I'll be doing that moving forward

File: .eslintrc.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-  },
-  parserOptions: {
+	root: true,
+	env: {
+		browser: true,
+		node: true
+	},
+	parserOptions: {
 	parser: 'babel-eslint'
 	// "parser": "babel-eslint"
```
[the remark this answers is not in the document]
Comment (TeddTech) [reply .eslintrc.js]: This comment has been removed

File: .eslintrc.js [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-  rules: {
-	  "indent": [2, "tab"],
-	  "no-tabs": 0,
-	  "quotes": [2, "double", { "avoidEscape": true }],
-	  "import/no-named-as-default": 0
-  }
+		'@nuxtjs',
+		'plugin:nuxt/recommended'
+	],
+	// add your custom rules here
```
[the remark this answers is not in the document]
Comment (TeddTech) [reply .eslintrc.js]: This comment has been removed

File: components/Header.vue [hunk 3]
```diff
@@ -89,74 +89,74 @@ export default {
...
 
 }
 
 .nav-sidebar ul li a {
-    text-align: left;
+		text-align: left;
 	font-weight: 500;
-    font-size: 22px;
+		font-size: 22px;
 }
```
[the remark this answers is not in the document]
Comment (TeddTech) [reply components/Header.vue]: double tabs have been removed

File: package.json [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
+	"version": "1.0.0",
+	"description": "We Count Project",
+	"author": "Ted Thompson",
+	"private": true,
+	"scripts": {
+		"dev": "nuxt",
+		"build": "nuxt build",
+		"start": "nuxt start",
+		"generate": "nuxt generate",
+		"lint": "eslint --ext .js,.vue --ignore-path .gitignore .; exit 0",
```
[the remark this answers is not in the document]
Comment (TeddTech) [reply package.json]: the exit code has been removed

File: .eslintrc.js [hunk 5]
```diff
@@ -1,25 +1,24 @@
...
+		node: true
+	},
+	parserOptions: {
 	parser: 'babel-eslint'
-	// "parser": "babel-eslint"
-  },
-  extends: [
+	},
+	extends: [
 	//'eslint-config-fluid',
```
Comment (greatislander) [inline .eslintrc.js]: This comment should also be removed.

File: .eslintrc.js [hunk 6]
```diff
@@ -1,25 +1,24 @@
...
-    node: true
-  },
-  parserOptions: {
+	root: true,
+	env: {
+		browser: true,
+		node: true
+	},
+	parserOptions: {
 	parser: 'babel-eslint'
```
Comment (greatislander) [inline .eslintrc.js]: Should be indented with two tabs.

File: .eslintrc.js [hunk 7]
```diff
@@ -1,25 +1,24 @@
...
-	  "indent": [2, "tab"],
-	  "no-tabs": 0,
-	  "quotes": [2, "double", { "avoidEscape": true }],
-	  "import/no-named-as-default": 0
-  }
+		'@nuxtjs',
+		'plugin:nuxt/recommended'
+	],
+	rules: {
+		"indent": [2, "tab"],
```
Comment (jobara) [inline .eslintrc.js]: I'm not sure if this is not working or I'm misunderstanding something. It seems that you are using a single tab and not 2 tabs for indentation. Does the 2 mean something else?

File: .eslintrc.js (hunk 5 above)
Comment (TeddTech) [reply .eslintrc.js]: This comment has been removed

File: .eslintrc.js (hunk 6 above)
Comment (TeddTech) [reply .eslintrc.js]: This has been corrected

File: .eslintrc.js (hunk 7 above)
Comment (TeddTech) [reply .eslintrc.js]: I believe you are right it should be one.

This change has been added

**Pull request merged**